### PR TITLE
New Chip component

### PR DESCRIPTION
**Pull request by trcywu**
## Screenshot / video

Primary: 
image

Secondary:
image

## What does this do?
Figma: https://www.figma.com/file/FqSaizGOyOzXFZNmZ4X0LGMn/%F0%9F%8D%AC-S'mores?node-id=2635%3A2301

- New Chip component
- Primary and secondary state, always with an icon
- Fixed width

### CODE REVIEW

File: src/Chip/Chip.tsx [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+
+export const Chip: FC<Props> = ({
+  children,
+  handleClick,
+  primary = false,
+  secondary = false,
+  icon = '',
+}) => {
+  return (
+    <div>
```
Comment (fleonard) [inline src/Chip/Chip.tsx:29]: Do we need this extra div?

**Review by fleonard — CHANGES_REQUESTED** [review]
Code looks good, just a couple of minor things missing for release:
- Can you update the CHANGELOG file with the release notes
- Can you add the storybook link to the Chip component in the README

File: src/Chip/Chip.tsx (hunk 1 above)
Comment (trcywu) [reply src/Chip/Chip.tsx:29]: we do not 👍

**Review by habin-isa — APPROVED** [review]
Looks great :shipit:

File: CHANGELOG.md [hunk 2]
```diff
@@ -29,3 +29,15 @@
...
+### Changed
+- tertiary button styling
+
+### Added
+- bg4 colour to theme
+
+## [0.65.0] - 2021-05-04
+### Added
+- New Chip component
+
```
Comment (fleonard) [inline CHANGELOG.md:43]: Can you move these at the top to maintain the order of the changelog?
Latest updates should be added at the top of the file.

Also can you add the new release link above 0.63.0? This is needed to properly link the changelog file to the github changes page. Should looks like:
`[0.65.0]: https://github.com/marshmallow-insurance/smores-react/compare/v0.64.0...v0.65.0`

**Review by fleonard — CHANGES_REQUESTED** [review]
Couple of minor things in the changelog, I should have run you through it before. Sorry didn't had enough time, hope it's more clear now with all my comments 😄

File: CHANGELOG.md (hunk 2 above)
Comment (trcywu) [reply CHANGELOG.md:43]: No problem! Thanks for the instructions

Comment (trcywu) [reply CHANGELOG.md:43]: Do I add the new release link after my PR is merged? As I haven't created v0.65.0 yet, right?

**Review by fleonard — APPROVED** [review]

**Pull request merged**